pkg/hookdeck/client_test.go

```diff
@@ -31,7 +33,16 @@ func TestPerformRequest_ParamsEncoding_Delete(t *testing.T) {
 	params.Add("key_a", "value_a")
 	params.Add("key_b", "value_b")
 
-	resp, err := client.PerformRequest(context.TODO(), http.MethodDelete, "/delete", params.Encode(), nil)
+	req := &http.Request{
+		Method: http.MethodDelete,
+		URL: &url.URL{
+			Scheme:   baseURL.Scheme,
+			Host:     baseURL.Host,
+			Path:     "/delete",
+			RawQuery: params.Encode(),
+		},
+	}
+	resp, err := client.PerformRequest(context.TODO(), req)
```

Comment:
There's a bit of duplication across these request setups and reusing the `baseURL` but I didn't think it was worth the time / effort to attempt a refactor since they were already broken I just wanted to get them to a passing state.